python/lsst/obs/base/ingest_tests.py

```diff
@@ -160,25 +160,62 @@ def checkRepo(self, files=None):
         """
         pass
 
+    def _createRepo(self):
+        """Use the Click `testing` module to call the butler command line api
+        to create a repository."""
+        runner = click.testing.CliRunner()
+        result = runner.invoke(butlerCli, ["create", self.root])
+        self.assertEqual(result.exit_code, 0, f"output: {result.output} exception: {result.exception}")
+
+    def _ingestRaws(self, transfer):
+        """Use the Click `testing` module to call the butler command line api
+        to ingest raws.
+
+        Parameters
+        ----------
+        transfer : `str`
+            The external data transfer type.
+        """
+        runner = click.testing.CliRunner()
+        result = runner.invoke(butlerCli, ["ingest-raws", self.root,
+                                           "--output-run", self.outputRun,
+                                           "--file", self.file,
+                                           "--transfer", transfer,
+                                           "--ingest-task", self.rawIngestTask])
+        self.assertEqual(result.exit_code, 0, f"output: {result.output} exception: {result.exception}")
```

Comment:
You had mentioned to mark the `script` functions as private api. I thought it would be better to just use the butler command in those locations instead, and get the extra mileage on testing it. (Let me know if you disagree)

Reply:
This works for me.